docs/app-check/overview.mdx

```diff
@@ -41,9 +41,30 @@ $ flutter run --no-sound-null-safety
 For legacy package imports, place the following ignore comment to hide Dart analyzer warnings:
 
 ```dart
+// ignore: import_of_legacy_library_into_null_safe
 import 'package:firebase_app_check/firebase_app_check.dart';
 ```
 
+### Activating the default provider
```

Comment:
@rbrishabh, the code example is already documented [here](https://firebase.flutter.dev/docs/app-check/usage#activating-the-default-provider). Please remove. Thanks.